Add p90 and p99 to the executor queue length graph

iain-macdonald opened the pull request.

maggie-lou commented:
I typically look at the graph named "Executor auto scaling" under "Executor pool" because it also has info on pods running and autoscaler target. Might be helpful to add these to that graph too?

iain-macdonald commented:
I went ahead and removed the "Executor queue length" graph and just added the p90 and p95 to the "Executor auto scaling" graph like you recommended (though I kept the average and median). Let me know if that sounds good to you? It looks like this now:

![Screenshot 2025-05-29 at 4 29 48 PM](https://github.com/user-attachments/assets/89396a02-245e-4591-80ba-3b2bad4fcd98)
